jgrapht-core/src/main/java/org/jgrapht/Graphs.java

```diff
@@ -576,6 +580,60 @@
     {
         return !graph.incomingEdgesOf(vertex).isEmpty();
     }
+
```

Comment:
Oh, for the wrapper, I was referring to a version that takes a Graph as input and produces the mapping for the Graph's vertexSet.  That's the most common use case in algorithms, and it is also something we want to to be able to optimize later for graphs which already maintain the mapping internally.

Reply:
@jsichi Yeah, I thought for a while why three wrapper functions would be needed in `Graphs`. I haven't thought for a second that the wrapper should actually take the graph for which it computes the mapping. :)